modules/widgets/contact-info.php

```diff
@@ -123,6 +124,10 @@ function widget( $args, $instance ) {
 				}
 			}
 
+			if ( '' != $instance['email'] ) {
```

Comment:
Imo, the reason that we should display the text if it's not an email, is that you can add multiple email addresses or some text before or after the email.